src/app/loadout-builder/filter/TierSelect.tsx

```diff
@@ -77,7 +76,7 @@ export default function TierSelect({
                     {statRangesFiltered
                       ? t('LoadoutBuilder.MaxTier', {
                           tier: t('LoadoutBuilder.TierNumber', {
-                            tier: statTierWithHalf(statRangesFiltered[statHash].max),
+                            tier: statRangesFiltered[statHash].max,
```

Comment:
One small change is that this now returns full tiers only, not stat points (thus also not half tiers). But with the new UI only full reachable tiers are indicated anyway.